Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -219,7 +219,15 @@ internal final class RewardCell: UITableViewCell, ValueCell {
     self.viewModel.outputs.cardViewDropShadowHidden
       .observeForUI()
       .observeValues { [weak self] hidden in
-        self?.cardView.layer.shadowOpacity = hidden ? 0 : 1
+        self?.cardView.layer.shadowOpacity = hidden ? 0 : 0.17
+    }
+
+    self.viewModel.outputs.cardViewBorder
+      .observeForUI()
+      .observeValues { [weak self] _ in
+        self?.cardView.layer.borderColor = UIColor.ksr_grey_400.cgColor
+        self?.cardView.layer.borderWidth = 1.0
+        self?.cardView.layer.cornerRadius = 2.0
```

Comment:
So I think we need to consider what happens if a reward goes from all-gone to not all-gone (which could happen if a backer cancels a pledge - I _believe_). This output should really be toggling the visibility of the border because if the reward becomes available, it won't change the view. Also, I think this border could remain on reusing the cell...? The easiest resolution is to set the `borderWidth` to `0.0` when this output emits to hide the border (assuming you will rename this output to `cardViewBorderIsHidden` or `cardViewBorderIsVisible` ;).